src/components/oInput.js

```diff
@@ -188,7 +226,7 @@ oInput.prototype.placeholder = function(placeholder) {
 }
 
 oInput.prototype.readOnly = function(readonly) {
```

Comment:
```suggestion
oInput.prototype.readonly = function(readonly) {
```
Może dajmy to też z małej litery, to jeden wyraz jest. W atrybutach też wszystko z małych podajesz

Reply:
napisałem wszystko, ale chodziło mi ten atrybut `readonly`